david_vallejos/tic_tac_toe/.gitignore

```diff
@@ -0,0 +1,30 @@
+# Compiled class file
```

Comment:
This task is about space_challenge, please remove all the files that are not related with it.